extension/js/content_scripts/webmail/gmail-element-replacer.ts

```diff
@@ -372,6 +373,7 @@ export class GmailElementReplacer implements WebmailElementReplacer {
     let msgEl = this.getMsgBodyEl(msgId); // not a constant because sometimes elements get replaced, then returned by the function that replaced them
     const senderEmail = this.getSenderEmail(msgEl);
     const isOutgoing = !!this.sendAs[senderEmail];
+    $(this.sel.attachmentsButtons).hide();
```

Comment:
So these buttons are hidden if `processAttachments` is called at least once?
and it is called when an attachment file name fits the pattern of `webmailNamePattern`
However, in PR#4202 we're adding a filename masked as `ATTNNNNN` to support Outlook email encrypted attachments, my vision is that `webmailNamePattern` is a rough recognition mechanism, meaning that not necessarily  every file named `ATTNNNNN` is encrypted. Maybe we should be thinking of disabling these buttons if a file is actually recognized as encrypted (or signed) by treatAs (and the like), not by triggering the recognbition process..

Reply:
ping @tomholub

Reply:
Yes, that is very sensible to only remove these when we actually recognize an encrypted message. 

Roman - could you please update the logic accordingly? Limonte moved on to another company at the end of last year.